<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>客户端</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <family>宋体</family>
           </font>
          </property>
          <property name="title">
           <string>操作栏</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btnAdd">
             <property name="font">
              <font>
               <family>宋体</family>
              </font>
             </property>
             <property name="text">
              <string>添加文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddDir">
             <property name="font">
              <font>
               <family>宋体</family>
              </font>
             </property>
             <property name="text">
              <string>添加文件夹</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSend">
             <property name="font">
              <font>
               <family>宋体</family>
              </font>
             </property>
             <property name="text">
              <string>发送文件</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <family>宋体</family>
           </font>
          </property>
          <property name="title">
           <string>文件列表(右键可删除)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QListWidget" name="listWidget">
             <property name="font">
              <font>
               <family>宋体</family>
              </font>
             </property>
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectItems</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="font">
           <font>
            <family>宋体</family>
           </font>
          </property>
          <property name="title">
           <string>发送进度</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="font">
              <font>
               <family>宋体</family>
              </font>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>230</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
         </font>
        </property>
        <property name="title">
         <string>局域网地址列表</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QComboBox" name="hostBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnFlushAddrList">
           <property name="font">
            <font>
             <family>宋体</family>
            </font>
           </property>
           <property name="text">
            <string>刷新列表</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="theme">
    <property name="title">
     <string>主题</string>
    </property>
    <addaction name="actionDark"/>
    <addaction name="actionLightblue"/>
    <addaction name="actionFlatwhite"/>
    <addaction name="actionUbuntu"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionLogDir"/>
   </widget>
   <addaction name="theme"/>
   <addaction name="help"/>
  </widget>
  <action name="actionDark">
   <property name="text">
    <string>dark</string>
   </property>
  </action>
  <action name="actionLightblue">
   <property name="text">
    <string>lightblue</string>
   </property>
  </action>
  <action name="actionUbuntu">
   <property name="text">
    <string>Ubuntu</string>
   </property>
  </action>
  <action name="actionFlatwhite">
   <property name="text">
    <string>flatwhite</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="actionLogDir">
   <property name="text">
    <string>日志目录</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
